src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -79,15 +79,17 @@ class AutoImportFix(element: RsElement, private val type: Type) : LocalQuickFixO
             if (path.reference == null) return null
 
             val basePath = path.basePath()
-            if (!basePath.isUnresolved) return null
+            if (basePath.resolveStatus == PathResolveStatus.RESOLVED) return null
```

Comment:
Does this quick fix make sense in case of a malformed path?
Looks like we should use `!= PathResolveStatus.UNRESOLVED` here